### DISCUSSION TITLE
Cümlelerin kaçı doğru sorusu

### DISCUSSION THREAD

**Discussion by sonsuzus** [sonsuzus/sonsuzus#236] [Zeka Sorusu] [answered]
Aşağıdaki 7 cümle içinde;
A)hiçbirisi doğru değildir
B)doğru olanların sayısı 6dir
C)yanlış olanların sayısı 5tır
D)doğru olanların sayısı 4tur
E)yanlış olanların sayısı 3tur
F)doğru olanların sayısı 2dir
G)yanlış olanların sayısı 1dir
Bu cümlelerden hangisi/hangileri doğruyu söylüyor?

**Comment by ElifNurSoyler** [comment, marked as answer]
C ve F şıkları doğru. Diğer şıklar yanlış.
